.github/workflows/ci-scripts-build.yml

```diff
@@ -66,6 +66,8 @@ jobs:
                                   -Wno-error=sizeof-pointer-memaccess
                                   -Wno-error=nonnull
                                   -Wno-error=dangling-pointer
+                                  -Wno-error=format-overflow
+                                  -Wno-error=format-security
```

Comment:
Recured for https://github.com/epics-base/epics-base/blob/a74789d9c0e55f6499b66f99cf0a745c681d884f/modules/database/src/std/rec/printfRecord.c#L172
"format not a string literal and no format arguments"

Reply:
The printfRecord.c warning should be corrected by 1f2edb69d214c25f75e97bc07c3d3a2f2a103639.